mungegithub/mungers/approval-handler.go

```diff
@@ -363,6 +364,15 @@ func (h *ApprovalHandler) getMessage(obj *github.MungeObject, ownersMap map[stri
 	if isFullyApproved {
 		title = "This PR is **APPROVED**"
 	}
+	approverList := suggestedApprovers.List()
+	forMachine := make(map[string][]string)
```

Comment:
this would be shorter as `forMachine := map[string][]string{"approvers": suggestedApprovers.List()}